misc/vim/syntax/ql.vim

```diff
@@ -0,0 +1,42 @@
+syn keyword qlKeyword       from where select predicate in as order by asc desc module result this super
+syn keyword qlAnnotation    abstract cached external final library noopt private
```

Comment:
A few annotations are missing: `deprecated` `override` `query` `pragma` `language` `bindingset`. Also, if `noopt` is included then the other annotation args ought to be included as well: `noinline` `nomagic` `monotonicAggregates`.

Reply:
Thanks, added.